.github/workflows/deploy-dev.yaml

```
@@ -38,3 +38,11 @@ jobs:
           tags: ${{ env.OCIR_REGISTRY }}/${{ env.OCIR_NAMESPACE }}/${{ env.OCIR_REPOSITORY }}:${{ env.IMAGE_TAG }}
           build-args: |
             ENV=dev
+
+      - name: Update GitOps image tag
+        uses: wafflestudio/waffle-world-oci/.github/actions/update-image@main
+        with:
+          image: ${{ env.OCIR_REGISTRY }}/${{ env.OCIR_NAMESPACE }}/${{ env.OCIR_REPOSITORY }}
+          tag: ${{ env.IMAGE_TAG }}
+          overlay-path: argocd/${{ inputs.ocir_repository }}
```

actionlint (1.7.12)

[error] property "ocir_repository" is not defined in object type {}

(expression)